DEVELOP.rst

```diff
@@ -91,7 +109,7 @@ the Maven repository.
 
 However, if you'd like to do this manually, you can run::
 
-    $ ./gradlew uploadArchives
+    $ ./gradlew publishToSonatype closeAndReleaseSonatypeStagingRepository
```

Comment:
Shouldn't we mention here the:
`ORG_GRADLE_PROJECT_signingKey/Password`?

Reply:
Wouldn't be safer to add `clean` task first, to make sure that the jars are 100% built with the intended code?